docs/publishing.md

```diff
@@ -117,9 +117,14 @@ These tests live in `after-build-dist.test.js`.
 
 16. Open a PR for these changes.
 
-17. Run `lerna clean` to remove `node_modules` directory created in each package. 
+17. Run `lerna clean` to remove `node_modules` directory created in each package.
 As documented in [Lerna docs](https://github.com/lerna/lerna/#clean)
 
+18. Create PRs to update the GOV.UK Design System and the Prototype Kit to use the new release of GOV.UK Frontend. Move the Trello cards to done when finished.
```

Comment:
Can we setup greenkeeper to do this for us?

Reply:
can greenkeeper work with private npm packages?

Reply:
Yes but only when paid for, our packages should be open long term.

So not worth thinking about atm true.